server/raft.go

```diff
@@ -263,6 +263,7 @@ var (
 	errLeaderLen         = fmt.Errorf("raft: leader should be exactly %d bytes", idLen)
 	errTooManyEntries    = errors.New("raft: append entry can contain a max of 64k entries")
 	errBadAppendEntry    = errors.New("raft: append entry corrupt")
+	errChannelFull       = errors.New("raft: entry failed to be placed on internal channel")
```

Comment:
Are there other places we could use this? I think this code has a few select blocks on channels IIRC.

Reply:
In this file, we either use a different error, such as `errProposalFailed`, or simply n.error()/n.warn()/n.debug() in the "default:" case. Following our discussion about possibly removing use of channels, I think this is a stop gap and this error will disappear in the future.